src/js/components/CheckBox/stories/CheckBoxInsideButton.js

```diff
@@ -1,37 +1,29 @@
-import React, { Component } from 'react';
-import { storiesOf } from '@storybook/react';
+import { Box, Button, CheckBox, Grommet, Text } from 'grommet';
+import React, { useState } from 'react';
 
-import { Box, Button, Grommet, CheckBox, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
+import { storiesOf } from '@storybook/react';
```

Comment:
Why did we change the imports?
Can we please use the following order:
```
import React, { useState } from 'react';
import { storiesOf } from '@storybook/react';
import { Box, Button, Grommet, CheckBox, Text } from 'grommet';	
import { grommet } from 'grommet/themes';	
```

Reply:
Sorry about that, probably my VS Code extensions do that. I will fix it!